cmd/run/run.go

```diff
@@ -86,6 +87,7 @@ func init() {
 
 	flags.AddConfig(run.Flags(), &params.configFile)
 	run.Flags().StringVarP(&params.addr, "addr", "a", defaultLocalAddr, "Set listening address of the server")
+	run.Flags().StringVarP(&params.contextPath, "context-path", "", "", "Set context path (eg. /my/path) that will define root path of the server")
```

Comment:
Hmm. I kind of think that a config value would be more befitting? If you agree, let's add something like [this Service section](https://github.com/open-policy-agent/opa-control-plane/pull/124/files#diff-54c7c1af5fa8d5db4dc49f0e8e80e93ba2b1183ba4d5c9e2e5729e6deae6a3cdR44) (from another PR) and add `api_prefix` or something like that to it. I'd avoid "context" as it's a bit vaguer than needed, I think.